website/src/edit-word-panel.tsx

```diff
@@ -0,0 +1,127 @@
+import React from "react"
+import { GoPencil } from "react-icons/go"
+import { IoCheckmark } from "react-icons/io5"
+import { Button } from "reakit/Button"
+import {
+  unstable_Form as Form,
+  unstable_FormInput as FormInput,
+  unstable_FormLabel as FormLabel,
+} from "reakit/Form"
+import { unstable_FormSubmitButton as FormSubmitButton } from "reakit/Form"
+import {
+  buttonStyle,
+  cleanButtonStyle,
+  formInputLabelStyle,
+  formStyle,
+  inputStyle,
+  textareaStyle,
+} from "./edit-word-panel.css"
+import { useForm } from "./form-context"
+
+/** Button that allows user to enter edit mode in the word panel, and edit fields of a word. */
+export const EditButton = () => {
+  const { form, isEditing, setIsEditing } = useForm()
+
+  return (
+    <Form {...form} className={formStyle}>
+      {isEditing ? (
+        // Displays a "Cancel" button and "Save" button in editing mode.
+        <>
+          <Button
+            className={cleanButtonStyle}
```

Comment:
I would suggest using the component from our component library instead, looks like all you changed is the font family. If you must do that, you can do `<CleanButton className={xxx}` and your `xxx` class will _add_ to, not replace the base class for that component.

Reply:
Gotcha, hope it's okay but I went ahead and changed it to an `IconButton` so there will be a gray background when hovering over the button.